python-package/lets_plot/plot/geom_extras.py

```diff
@@ -16,23 +15,39 @@ def arrow(angle=None, length=None, ends=None, type=None):
 
     Parameters
     ----------
-    angle : numeric
-        The angle of the arrow head in degrees (smaller numbers produce narrower, pointier arrows). 
+    angle : int
+        The angle of the arrow head in degrees (smaller numbers produce narrower, pointer arrows). 
         Essentially describes the width of the arrow head.
-    length : numeric
+    length : int
         The length of the arrow head (px).
-    ends : ['last' \ 'first' | 'both']
+    ends : ['last' | 'first' | 'both']
         Indicating which ends of the line to draw arrow heads.
     type : ['open' | 'closed']
         Indicating whether the arrow head should be a closed triangle.
         
     Returns
     -------
-        arrow object specification
-        
+    `LayerSpec`
+        Arrow object specification.
+
     Examples
-    ---------
-    >>> from lets_plot import *
-    >>> ggplot() + geom_segment(aes(x=[3], y=[6], xend=[4], yend=[10]), arrow=arrow(type='closed'))
+    --------
+    .. jupyter-execute::
+        :linenos:
+        :emphasize-lines: 5, 7, 9, 11
+        
+        from lets_plot import *
+        LetsPlot.setup_html()
+        ggplot() + \\
+            geom_segment(aes(x=[3], y=[6], xend=[4], yend=[10]), \\
```

Comment:
When we have only one value for each aesthetic, we can just use them as parameters, like this:

```python
... + geom_segment(x=3, y=6, xend=4, yend=10, arrow=arrow(type='closed')) + ...
```